api/app/auto_spatial_advisory/process_hfi.py

```diff
@@ -26,25 +31,89 @@ class RunType(Enum):
     FORECAST = 'forecast'
     ACTUAL = 'actual'
 
+    @staticmethod
+    def from_str(label):
+        if label in ('forecast', 'Forecast', 'FORECAST'):
+            return RunType.FORECAST
+        elif label in ('actual', 'Actual', 'ACTUAL'):
+            return RunType.ACTUAL
+        else:
+            raise NotImplementedError
+
 
 class UnknownHFiClassification(Exception):
     """ Raised when the hfi classification is not one of the expected values. """
 
 
+def create_cloud_optimized_raster(source_dataset):
```

Comment:
this works in dev?!

Reply:
oh wait - I think this function just isn't ever getting called in this branch. Could just delete it.